CHANGELOG

```diff
@@ -25,6 +25,7 @@
     - High: (Authenticated|Unauthenticated) (admin|author|contributor) [vulnerability description]
     - Medium: (Authenticated|Unauthenticated) (admin|author|contributor) [vulnerability description]
     - Low: (Authenticated|Unauthenticated) (admin|author|contributor) [vulnerability description]
+    - Upgraded gems: time (CVE-2023-28756)
```

Comment:
We don't add dependencies in the Security Fixes section so just add `time` to the Upgraded Gems on line 5